README.md

```diff
@@ -217,6 +217,10 @@ function specified by the __contactSuccess__ parameter.
 - Windows 8
 - Windows
 
+### iOS Quirks
+
+- On iOS, contact object, passed to `contactSuccess` will also contain some properties, such as `phoneNumbers`, `emails` and `ims`, fetched from contacts, linked to one, which was picked by user. However these fields from linked contacts won't be saved to device's address book. This is made intentionally to prevent creating lots of fields duplicates in both unified and linked contact.
+
```

Comment:
The change fixes the behaviour of `pickContact` so it's reasonable to put this into iOS quirks for `pickContact`. However this possibly might affect the code that dealing with `save`, so IMO it is convenient to emphasise this there as well, so the end user won't need to search for this through the whole doc.

if you think that one of these sections is excess, i'll remove that.